service-graphql/src/main/resources/data.json

```diff
@@ -2849,9 +2849,9 @@
     },
     {
       "id": "afterparty",
-      "name": "Surprise",
-      "address": "Coming soon 🍻",
-      "imageUrl": "https://wallpapers.com/images/featured/1y550aojs77u58wk.jpg"
+      "name": "Café Oz",
```

Comment:
Est-ce que c'est possible d'ajouter les coordonnées aussi ? Pour la venue aussi si possible

Reply:
Yes it is! Let me check

Reply:
https://github.com/paug/AndroidMakersBackend/pull/14/commits/c9e6141c6d6957444d841e63947f2270386dcec6